src/uct/ib/base/ib_iface.h

```diff
@@ -107,6 +107,18 @@ enum {
     UCT_IB_ADDRESS_PACK_FLAG_PKEY          = UCS_BIT(5)
 };
 
+
+/**
+ * Reachability mode.
+ */
+typedef enum uct_ib_reachability_mode {
```

Comment:
@iyastreb What do you mean?

Reply:
since you have a typedef, the enum name is redundant:
```C
typedef enum {
} uct_ib_reachability_mode_t;
```
Then you access it everywhere by typedef, not by `enum uct_ib_reachability_mode`, so the enum name is actually redundant.